pkg/skaffold/deploy/helm.go

```diff
@@ -124,12 +126,15 @@ func (h *HelmDeployer) Deploy(ctx context.Context, out io.Writer, builds []build
 	return NewDeploySuccessResult(namespaces)
 }
 
+// Dependencies returns a list of files that the deployer depends on.
 func (h *HelmDeployer) Dependencies() ([]string, error) {
 	var deps []string
+
 	for _, release := range h.Releases {
-		deps = append(deps, release.ValuesFiles...)
+		r := release
```

Comment:
this seems unnecessary, but if you want to change the name, then do it in the line above:

`for _, r := range h.Releases {`

Reply:
This pinning was done to avoid a lint error about range scoped variables in a function literal (see `filepath.Walk` call below).

This appears to have been unimportant in this case.